src/main/scala/elo/LeaderboardService.scala

```diff
@@ -1,9 +1,28 @@
 package elo
+import scala.util.Try
 
 class LeaderboardService(playerService: PlayerService) {
 
-  def getLeaderboard(numTopPlayers: Int = 3): Seq[Player] = {
-    playerService.findAllPlayers.sortBy(-_.score).take(numTopPlayers)
+  val defaultLeaderboardSize = 3
+
+  def getLeaderboardSize(messageText: String): Option[Int] = {
```

Comment:
I'd argue that this method is more appropriate in some message parsing class/module, rather than in the leaderboard service. It's fine to leave here for now, but if you agree then let's slap a todo on this to move it somewhere in the future